### PR TITLE
Use a version of Hydra::Collections that uses PCDM

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: spec/actors/curation_concern/generic_work_actor_spec.rb [hunk 1]
```diff
@@ -170,6 +166,7 @@
         reload = GenericWork.find(curation_concern.id)
         expect(reload.identifier).to be_blank
         expect(reload).to be_persisted
+        pending "aggregation delete is not working so it's not removing from old collections. Possibly fixed by https://github.com/projecthydra-labs/activefedora-aggregation/pull/33"
```
Comment (cam156) [inline spec/actors/curation_concern/generic_work_actor_spec.rb]: Can you create a ticket for this pending so it does not get lost?

File: spec/views/my/_list_works.html.erb_spec.rb [hunk 2]
```diff
@@ -4,15 +4,17 @@
 
   let(:work_title) { 'Work Title' }
   let(:coll_title) { 'Collection Title' }
-  let(:collection) { FactoryGirl.create(:collection, title: coll_title) }
-  let!(:work) { FactoryGirl.create(:work, title: [work_title], collections: [collection]) }
+  let(:collection) { FactoryGirl.build(:collection, id: '3197z497t', title: coll_title) }
+  let!(:work) { FactoryGirl.build(:work, id: '3197z511f', title: [work_title]) }
```
Comment (cam156) [inline spec/views/my/_list_works.html.erb_spec.rb]: @jcoyne How does the work get assigned to the collection here?  what am I missing?

File: sufia-models/app/actors/curation_concern/generic_work_actor.rb [hunk 3]
```diff
@@ -46,7 +46,11 @@ def add_to_collections(new_collection_ids)
       end
 
       #add to new
-      curation_concern.collection_ids = new_collection_ids
+      new_collection_ids.each do |coll_id|
+        collection = Collection.find(coll_id)
+        collection.members << curation_concern
+        collection.save
```
Comment (cam156) [inline sufia-models/app/actors/curation_concern/generic_work_actor.rb]: @jcoyne Do we really need to save after each assign?  Couldn't this be moved outside the loop?

File: spec/actors/curation_concern/generic_work_actor_spec.rb (hunk 1 above)
Comment (jcoyne) [inline spec/actors/curation_concern/generic_work_actor_spec.rb]: https://github.com/projecthydra/sufia/issues/1170

File: spec/views/my/_list_works.html.erb_spec.rb (hunk 2 above)
Comment (jcoyne) [inline spec/views/my/_list_works.html.erb_spec.rb]: It doesn't in this test. This is a view test, so I just stubbed out the `CollectionMemberService` to return the correct stuff.

File: sufia-models/app/actors/curation_concern/generic_work_actor.rb (hunk 3 above)
Comment (jcoyne) [inline sufia-models/app/actors/curation_concern/generic_work_actor.rb]: Yes, we need to save each collection. There's no way to move it outside the loop.

**Pull request merged**